web/sites/default/config/views.view.content_updated_recently.yml

```diff
@@ -364,7 +364,7 @@ display:
           exclude: false
           alter:
             alter_text: true
-            text: '{{ revision_uid }} updated <strong>{{ type }}</strong> {{ title }} at {{ changed }}. <strong>Revision log:</strong> {{ revision_log }}'
+            text: '{{ revision_uid }} {{ ''updated''|t }} <strong>{{ type }}</strong> {{ title }} {{ ''at''|t }} {{ changed }}. <strong>{{ ''Revision log''|t }}:</strong> {{ revision_log }}'
```

Comment:
Hmm, this comment shows as "outdated" in the PR comments and I think you somehow reviewed an older version of this PR because that change has already been reverted and the file is no longer listed as a changed file in this PR.